create.php

```diff
@@ -87,6 +87,23 @@
         if ($db->query($schema))
             echo "Created table 'clients'.\n";
 
+        $schema = 'CREATE TABLE sysinfo (
+                    system_id STRING PRIMARY KEY,
```

Comment:
If you make the system id the primary key, only the last submitted data will be stored in the db. Is that your intention? Personally, I'd go for a composite key of (system_id, updated), or a db-managed auto-incrementing primary key.